testsuite/integration/basic/pom.xml

```diff
@@ -1363,9 +1363,13 @@
                                     </plugin-options>
                                     <feature-packs>
                                         <feature-pack>
-                                            <groupId>${testsuite.ee.galleon.pack.groupId}</groupId>
-                                            <artifactId>${testsuite.ee.galleon.pack.artifactId}</artifactId>
-                                            <version>${testsuite.ee.galleon.pack.version}</version>
+                                            <feature-pack>
```

Comment:
This adds another level of xml instead of just replacing the values in the 3 elements. This is why the Galleon job fails.

Reply:
Fixed, an oversight on my side since I did not test it locally before pushing it.